commons/passive/fitbit/fitbit_sleep_classic.avsc

```diff
@@ -0,0 +1,25 @@
+{
+  "namespace": "org.radarcns.passive.fitbit",
+  "type": "record",
+  "name": "FitbitSleepClassic",
+  "doc": "Classic sleep data as defined at https://dev.fitbit.com/build/reference/web-api/sleep/",
+  "fields": [
+        {
+            "name": "type",
+            "type": "string",
+            "doc": "Type of data coming from fitbit intraday API, (e.g. sleep classic)"
+        },{
+            "name": "level",
+            "type": "string",
+            "doc": "Level of sleep, 'awake', 'restless', or 'asleep'"
```

Comment:
If this is not expected to change, I'd propose to make this an enum instead
```json
"type": {
   "name": "SleepStageLevel",
   "type": "enum",
   "symbols": ["DEEP", "LIGHT", "REM", "AWAKE", "RESTLESS", "ASLEEP", "UNKNOWN"],
   "doc": "Sleep level."
},
"doc": "Level of sleep.",
"default": "UNKNOWN"
```
I don't know how `restless` compares to `light` sleep, are there duplicated definitions in this symbol list?

Reply:
Yes this is intentional since the fitbit API provides 2 measures of sleep based on processing data in 30 seconds interval and 60 seconds interval

Reply:
Quoting the Fitbit WEB API 
```
This endpoint supports two kinds of sleep data:

stages : Levels data is returned with 30-second granularity. 'Sleep Stages' levels include deep, light, rem, and wake.

classic : Levels data returned with 60-second granularity. 'Sleep Pattern' levels include asleep, restless, and awake.
```

Reply:
Perhaps, this one can be named `FitbitSleepPattern` then, and the other still `FitbitSleepStage`?